pkg/network/connection.go

```diff
@@ -70,6 +70,7 @@ type connection struct {
 	writeBuffer         *buffer.IoBufferPoolEntry
 	writeBufferMux      sync.RWMutex
 	writeBufferChan     chan bool
+	writeSchedChan      chan bool
```

Comment:
It's better to separate params between attachEventLoop and startRWLoop, and add some comments